api/src/main/java/com/github/streamshub/console/api/support/factories/ConsoleConfigFactory.java

```diff
@@ -143,4 +148,6 @@ private String resolveValue(String value) {
 
         return value;
     }
+
```

Comment:
shall we setup a checkstyle rule for more than one empty lines? i had issue with that in my PR as well